airbyte-integrations/connectors/source-shopify/source_shopify/source.py

```diff
@@ -270,13 +279,50 @@ def read_records(
         yield from self.filter_records_newer_than_state(stream_state=stream_state, records_slice=records)
 
 
+class MetafieldShopifySubstream(ShopifySubstream):
+    slice_key = "id"
+    data_field = "metafields"
+
+    parent_stream_class: object = None
+
+    def path(self, stream_slice: Mapping[str, Any] = None, **kwargs) -> str:
+        object_id = stream_slice[self.slice_key]
+        return f"{self.parent_stream_class.data_field}/{object_id}/{self.data_field}.json"
+
+
+class Articles(IncrementalShopifyStream):
+    data_field = "articles"
+
+    def path(self, **kwargs) -> str:
+        return f"{self.data_field}.json"
+
+
+class MetafieldArticles(MetafieldShopifySubstream):
+    parent_stream_class: object = Articles
+
+
+class Blogs(IncrementalShopifyStream):
```

Comment:
I think we are missing the `blogs` schema, please check the `schemas/blogs.json` for schema presence.